### DISCUSSION TITLE
host platform sleep/suspend in idle period of Openthread Border router

### DISCUSSION THREAD

**Discussion by aashu216** [openthread/openthread#9520] [Q&A] [answered]
Hi All, 

Is it possible to make FTD host platform sleep/suspend if no any network activity for from Openthread host for RCP based architecture without impacting any thread functionalities/specifications? host platform can be wakeup using external resource like interrupts.

**Comment by abtink** [comment, marked as answer]
Yes. This is technically possible ( most likely managed by underlying OS).

Some aspects to consider:
- How quickly can the host MCU resume from sleep/suspect on trigger (such as timer)? and if it would impact timers used within Thread stack. Some msec of extra delay may be ok for geneeral Thread operation but if device enables and uses CSL then this may impact CSL tx/rx.
- OT stack FTD device use a 1 second periodic timer (`TimeTicker`) which is used for a bunch of different purposes. So this should be taken into account, i.e., host need to be awakened at least once per second to process this.
